learn/tanssi/external-security-providers/symbiotic.md

```diff
@@ -0,0 +1,101 @@
+            curators["Vault Curators"]
+            resolvers["Resolvers"]
+            restakers -- Deposit Assets --> vaults
+            curators -- Manage --> vaults
+            slash@{ shape: bolt, label: "Communication link" }
+            resolvers -- Decide On Slashing Events --> slash
+            slash -- Executes On --> vaults
+        end
+    ```
+
+- **Operators** - are the computational component validating the transactions of the networks
+
+    ```mermaid 
+    flowchart TD
+        subgraph Validators
+            validators["Operators/Validators"]
+            operators["Node Operators"]
+            operators -- Run --> validators
+        end
+    ```
+
+- **Networks** - are the actively validated services or appchains. These application-specific blockchains can be a use case from a wide range of industries, such as Gaming, Defi, RWAs, and others, and are the platforms that, through dApps, the end users interact with
+
+
+```mermaid 
+    flowchart TD
+    subgraph Networks
+        networks["Networks/Appchains"]
+        developers["Developers"]
+        developers-- Launch --> networks
+    end
+```
+
+Around these components, different actors participate:
+
+- **Restakers** - are the ones providing liquidity to the vaults, obtaining rewards. Restakers choose which vaults they restake to, provided that the vault accepts their assets and they agree with the networks the vault works with and the general setup
+- **Vault Curators** - are responsible for the vault administration. Curators decide which networks the vault works with, which operators are whitelisted to participate in securing the networks, which assets are accepted as collaterals and the slashing and rewarding mechanisms
+- **Resolvers** - are responsible for resolving veto-slashing events in the vault. Veto slashing events are a particular type of event that requires the participation of resolvers, who have the authority to revoke the slashing request
+- **Node Operators** - are the ones running the actual operators, which validate the networks' transactions. Node operators are responsible for the configuration and hardware of the nodes. They apply to offer their services in both vaults and networks and have to be accepted and whitelisted by both before starting to validate networks and get rewards
+- **Developers** - are the ones building appchains
+
+### Tanssi with Symbiotic {: #tanssi-symbiotic }
+
+Developers launching appchains through Tanssi benefit from [block production services](/learn/tanssi/appchain-services/block-production/){target=\_blank} and the shared security model derived from every vault opting-in to support the Tanssi protocol. This eliminates the hurdle of dealing with infrastructural components developers would need to take on otherwise.
+
+Curators running vaults can apply to offer the restaked collaterals as economic security for the Tanssi network, regardless of how many appchains are running through the Tanssi protocol.
```

Comment:
I thought that curators could whitelist specific vaults they wanted to back. But I think that is not that case now that I think so right?

Reply:
We should still clarify a bit more this. Go through the FAQ document and provide some more insights here

Reply:
Going through the symbiotic docs, I think curator and managers are pretty much synonyms